Src/SmtpServer.Tests/SmtpServerTests.cs

```diff
@@ -271,7 +271,7 @@ public void SecuresTheSessionWhenCertificateIsSupplied()
             ServicePointManager.ServerCertificateValidationCallback = null;
         }
 
-        [Fact]
+        [Fact(Skip = "fix this currently not working with github build job")]
```

Comment:
do you know why these are working? do you think its something that we can get working?

Reply:
Currently a certificate is linked that does not exist in the code. Creating dynamically is difficult, in my tests it did not want to allow the self-signed certificate.


```cs
var certificate = File.ReadAllBytes(@"C:\Users\caino\Dropbox\Documents\Cain\Programming\SmtpServer\SmtpServer.pfx");
var password = File.ReadAllText(@"C:\Users\caino\Dropbox\Documents\Cain\Programming\SmtpServer\SmtpServerPassword.txt");

return new X509Certificate2(certificate, password);
```